tests/test_testbench_retry.py

```diff
@@ -655,6 +655,107 @@ def test_retry_test_return_error_after_bytes(self):
         self.assertIn("size", create_rest)
         self.assertEqual(int(create_rest.get("size")), 2 * UPLOAD_QUANTUM)
 
+    def test_write_retry_test_stall_after_bytes(self):
```

Comment:
Nice test. Can we add one for single shot uploads as well?

Reply:
Added